ex2/ex2.js

```diff
@@ -8,3 +8,25 @@
  *     fn(",,,", 4) === -1
  *     fn("hel,lowo,rld", 0) - Throws
  */
+function getTheCommaIndexAt(str, val) {
+	var index = -1, returnVal = 'Error';
+	for (var i = 0; i < val; i++) {
+		index = str.indexOf(',',index+1);
+		if (i === val-1 && index > -1) {
+			return index;
```

Comment:
The two return statements are a bit redundant. A positive value (also known as "the happy path") is returned here and not at the bottom. Could you refactor your code so that either 1) there's only one return statement that handles all return values. 2) there are two return statements, but it's clear what each one is in charge of.